lib/cppcheck.cpp

```diff
@@ -1033,22 +1041,30 @@ unsigned int CppCheck::checkFile(const std::string& filename, const std::string
     return mExitCode;
 }
 
-// TODO: replace with ErrorMessage::fromInternalError()
-void CppCheck::internalError(const std::string &filename, const std::string &msg)
+static ErrorMessage makeError(const std::string &filename, const std::string &msg, const std::string &id)
 {
     const std::string fullmsg("Bailing out from analysis: " + msg);
```

Comment:
I am not sure if we want to write "Bailout out from analysis" when an addon is not found. The whole cppcheck analysis is executed but the addon is skipped.